README.md

```diff
@@ -1,45 +1,43 @@
-# Adobe I/O Documentation Template
+# Adobe Commerce Developer Documentation
 
-This is a site template built with the [Adobe I/O Theme](https://github.com/adobe/aio-theme).
+Welcome! This site contains the latest Adobe Commerce and Magento Open Source developer documentation for ongoing releases of both products. For additional information, see our [Contribution Guide](https://developer.adobe.com/commerce/contributor/).
 
-View the [demo](https://adobedocs.github.io/dev-site-documentation-template/) running on Github Pages.  
+## Contributors
 
-## Where to ask for help
+Our goal is to provide the Adobe Commerce and Magento Open Source communities with comprehensive and quality technical documentation. We believe that to accomplish that goal we need experts from the community to share their knowledge with us and each other. We are thankful to all of our contributors for improving the documentation.
 
-The slack channel #adobeio-onsite-onboarding is our main point of contact for help. Feel free to join the channel and ask any questions.
+![Commerce contributors](https://raw.githubusercontent.com/wiki/magento/magento2/images/dev_docs_contributors.png)
 
-## How to develop
+## Local development
 
-For local development, simply use :
+This is a [Gatsby](https://www.gatsbyjs.com/) project that uses the [Adobe I/O Theme](https://github.com/adobe/aio-theme).
 
-```bash
-$ npm install
-$ npm run dev
-```
+To build the site locally:
 
-For the documentation developer, please read these sections on how to:
+1. Clone this repo.
+1. Install project dependencies.
 
-- [Arrange the structure content of your docs](https://github.com/adobe/aio-theme#content-structure)
-- [Linking to pages](https://github.com/adobe/aio-theme#links)
-- [Using assets](https://github.com/adobe/aio-theme-aio#assets)
-- [Setting Global Navigation](https://github.com/adobe/aio-theme#global-navigation)
-- [Setting Side Navigation](https://github.com/adobe/aio-theme#side-navigation)
-- [Using content blocks](https://github.com/adobe/aio-theme#jsx-blocks)
-- [Notes on using Markdown](https://github.com/adobe/aio-theme#writing-enhanced-markdown)
+   ```bash
+   yarn install
+   ```
 
-For more in-depth [instructions](https://github.com/adobe/aio-theme#getting-started).
+1. Build the project.
 
-## How to deploy
+   ```bash
+   yarn dev
```

Comment:
`yarn dev` doesn't build the project entirely. `yarn build` does. I use `yarn start` to preview locally, it combines `build` and `serve`. I agree that for development, `yarn dev` works better. So maybe change the description to something like _Launch the project in the development mode_

> gatsby develop will run at most 3 page queries (index page, actual 404 and develop 404) initially. The rest of the queries will run when they are needed (when the browser requests them). In contrast, gatsby build will run page queries for every page that doesn’t have cached and up to date results already.

https://www.gatsbyjs.com/docs/conceptual/overview-of-the-gatsby-build-process/#differences-between-develop-and-build

Reply:
I agree. I have a little different set of instructions in [Cloud tools readme](https://github.com/AdobeDocs/commerce-cloud-tools#available-commands) for this, such as using `yarn build`, but I could probably add a little more explanation.